e2e_samples/dataset_versioning/insert_sql/main.py

```diff
@@ -0,0 +1,61 @@
+import argparse
+import pandas as pd
+import sys
+
+from keyvault_wrapper import KeyvaultWrapper
+from sql_wrapper import SqlWrapper
+from process import Process
+
+
+def read_csv(path: str) -> pd.DataFrame:
+    """
+    Read csv and modify datatype
+
+    :param path: path to csv
+    :type path: str
+    :return: dataframe with required columns
+    :rtype: pd.DataFrame
+    """
+    df = pd.read_csv(path)
+    df = df[["id", "loan_amnt", "annual_inc", "dti", "delinq_2yrs", "total_acc","total_pymnt", "issue_d", "earliest_cr_line", "loan_status"]]
```

Comment:
Just a question. 
You intentionally convert `issue_d` type to date, but doesn't convert others. Do other column automatically covert data type ? (ex. `id` as int)

Reply:
others did not have real impact when adding to sql. So I did not specifically convert into different type. Datetime was not possible to refer automatically.